### PR TITLE
syncthingtray: 0.9.1 -> 0.10.0

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Other notes

Both #68277 and #68276 need to be merged for this to work.

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build syncthingtray-minimal  syncthingtray

**Comment by doronbehar** [comment]
As I said, Both #68277 and #68276 need to be merged for the build to work.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -20645,7 +20645,7 @@ in
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
   syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray { };
-  syncthingtray-minumal = libsForQt5.callPackage ../applications/misc/syncthingtray {
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray {
```
Comment (worldofpeace) [inline pkgs/top-level/all-packages.nix]: when was this attribute added? If it's in 19.09 like this we need to backport a change.
And I guess setup an alias, which is unfortunate if anyone has that in their config.

Comment (doronbehar) [reply pkgs/top-level/all-packages.nix]: This was committed by me (`git blame` me :point_left::) in https://github.com/NixOS/nixpkgs/pull/65480.

Comment (worldofpeace) [reply pkgs/top-level/all-packages.nix]: ah ok, no worries. If you make it a separate commit and add an alias to `aliases.nix` I can backport it for you.

Comment (doronbehar) [reply pkgs/top-level/all-packages.nix]: If you are talking about backporting the rename for me personally (as it seems I may be the only Nix user of this package) then I don't mind it at all - I can just change it in my own `nix-env`.

As for other users who may have come to rely on that, I think this change won't be too bad. We've already performed such renames, e.g https://github.com/NixOS/nixpkgs/pull/65724 .

**Review by ck3d — APPROVED** [review]

**Comment by worldofpeace** [comment]
Fixed the misspelled name in 548d0b73f2b8d0096b2e673219705a4abb0a0942

**Pull request merged**